1.9/overview/architecture/boot-sequence.md

```diff
@@ -0,0 +1,44 @@
+---
+post_title: Boot Sequence
+nav_title: Boot Sequence
+menu_order: 6
+---
+
+During DC/OS installation, the components come online in a particular sequence due to interdependencies.
```

Comment:
Suggest `During installation, the DC/OS components come online in a particular sequence because of interdependencies.`